components/ClientInfo/ClientInfo.tsx

```diff
@@ -0,0 +1,146 @@
+import styles from './ClientInfo.module.css';
```

Comment:
the styles are showing up but the reference in the styling sheet is saying it can't be found